lib/Utils/Logging.pm

```diff
@@ -205,6 +205,11 @@ sub export_logs {
     if ($utils::IN_ZYPPER_CALL) {
         upload_solvertestcase_logs();
     }
+
+    my $audit_log = "/var/log/audit/audit.log";
+    if (script_run("! [[ -e $audit_log ]]")) {
+        upload_logs("$audit_log");
+    }
```

Comment:
Maybe
```suggestion
    my $audit_log = "/var/log/audit/audit.log";
        upload_logs("$audit_log", failok => 1);
```

Reply:
Ok, I will update the code. Thanks for the review!